controllers/spec/enforcer/statefulset_spec/CustomMetadataSpecEnforcer.go

```diff
@@ -0,0 +1,125 @@
+/*
+Copyright 2021 Reactive Tech Limited.
+"Reactive Tech Limited" is a company located in England, United Kingdom.
+https://www.reactive-tech.io
+
+Lead Developer: Alex Arica
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
```

Comment:
just curious, do we need to add these headers for our custom fork patches?

Reply:
I don't think so, it's due to a copy-paste action xD